cypress.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    supportFile: false,
```

Comment:
This was added to address a Cypress warning. We currently don't use a support file for global setups or custom commands, so we set `supportFile` to `false`.